backend/user-service/src/main/java/com/speakapp/userservice/mappers/AppUserMapper.java

```diff
@@ -0,0 +1,21 @@
+package com.speakapp.userservice.mappers;
+
+import com.speakapp.userservice.dtos.*;
+import com.speakapp.userservice.entities.AppUser;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
+
+@Mapper
+public interface AppUserMapper {
+    AppUser appUserFromAppUserCreateDTO(AppUserCreateDTO appUserCreateDTO);
```

Comment:
Dużo warningów wyrzucają te metody, tutaj akurat przydałoby się stała wartość, żeby pola, które nie były nullami. Zmieniłem już to na branchu, zaraz to wrzucę, żebyś mógł sobie zobaczyć o co mi chodziło